plugins/catalog-react/package.json

```diff
@@ -9,7 +9,8 @@
   "publishConfig": {
     "access": "public",
     "main": "dist/index.esm.js",
-    "types": "dist/index.d.ts"
+    "types": "dist/index.d.ts",
+    "alphaTypes": "dist/index.alpha.d.ts"
```

Comment:
Also needs `--experimental-type-build` for the `"build"` script, and `"alpha"` added to `"files"`